src/shared/sleep-config.c

```diff
@@ -126,6 +131,26 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         return 0;
 }
 
+/* Freeze or thaw user.slice */
+int freeze_thaw_user_slice(const char **method) {
```

Comment:
why a ptr to a ptr? A simple ptr would suffice?

Reply:
ah, i understand why you are doing this. but please add a comment about it.

Reply:
Sure